Add FAQ entry on how to get WatchOS auth working (#118)
maxgoedjen/secretive: FAQ.md

Please run `ssh -Tv git@github.com` in your terminal and paste the output in a [new GitHub issue](https://github.com/maxgoedjen/secretive/issues/new) with a description of your issue.

> ### Secretive prompts for password input instead of using my Apple Watch

Reviewer: ```suggestion
### Secretive prompts me to type my password instead of using my Apple Watch
```

1) Make sure you have enabled "Use your Apple Watch to unlock apps and your Mac" in System Preferences --> Security & Privacy:

![System Preferences Setting](assets/apple_watch_system_prefs.png)

> 2) Ensure that unlocking your Mac with Apple watch is working (lock and unlock at least once)

Reviewer: ```suggestion
2) Ensure that unlocking your Mac with Apple Watch is working (lock and unlock at least once)
```

> 3) Now you should get prompted on the watch when you use `ssh` or `git fetch <remote_name>` etc. Double click the side button to approve:

Reviewer: ```suggestion
3) Now you should get prompted on the watch when your key is accessed. Double click the side button to approve:
```
